File: by_hand/lib/models/package.dart
class Package{
  int id;
  String name;
  String price;
  String description;

  Package();

  Package.fromJson(Map<String,dynamic>json){
    this.id=json['id'];
    this.name=json['name'];
    this.price=json['price'];
    this.description=json['description'];
  }
}

File: by_hand_api/src/models/package.js
import {  INTEGER, Model } from 'sequelize';

export default (sequelize, DataTypes) => {
  class Package extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here

    }
  };
  Package.init({
   
   
    name:DataTypes.STRING,
    price:DataTypes.STRING,
    description:DataTypes.STRING,

    deletedAt: 'destroyTime'
  }, {
    sequelize,
    tableName:'package',
    paranoid: true,
    modelName: 'Package',
    
  });
  return Package;
};